ecorest/config/main.go

```diff
@@ -2,19 +2,49 @@ package config
 
 import (
 	"github.com/OpenTreeMap/otm-ecoservice/eco"
+	"os"
 )
 
+type server struct {
+	Host string
+	Port string
+}
+
+type data struct {
+	Path string
+}
+
 type Config struct {
 	Database eco.DBInfo
-	Data     struct {
-		Path string
-	}
-	Server struct {
-		Host string
-		Port string
+	Data     data
```

Comment:
Do we even need the `Server` and `Data` structs above? Seems like they could just be fields in `Config`.